.circleci/config.yml

```diff
@@ -498,15 +498,22 @@ jobs:
           name: Tag images
           command: |
             docker images fundocker/marsha
-            DOCKER_TAG=$(echo ${CIRCLE_TAG} | sed 's/^v//')
-            echo "DOCKER_TAG: ${DOCKER_TAG} (from Git tag: ${CIRCLE_TAG})"
-            docker tag marsha:${CIRCLE_SHA1} fundocker/marsha:latest
+            DOCKER_TAG=$([[ -z "$CIRCLE_TAG" ]] && echo $CIRCLE_BRANCH || echo ${CIRCLE_TAG} | sed 's/^v//')
+            RELEASE_TYPE=$([[ -z "$CIRCLE_TAG" ]] && echo "branch" || echo "tag ")
+            # Display either:
+            # - DOCKER_TAG: master (Git branch)
+            # or
+            # - DOCKER_TAG: 1.1.2 (Git tag v1.1.2)
+            echo "DOCKER_TAG: ${DOCKER_TAG} (Git ${RELEASE_TYPE}${CIRCLE_TAG})"
             docker tag marsha:${CIRCLE_SHA1} fundocker/marsha:${DOCKER_TAG}
             docker tag marsha:${CIRCLE_SHA1}-dev fundocker/marsha:${DOCKER_TAG}-dev
-            docker tag marsha:${CIRCLE_SHA1}-alpine fundocker/marsha:alpine
             docker tag marsha:${CIRCLE_SHA1}-alpine fundocker/marsha:${DOCKER_TAG}-alpine
             docker tag marsha:${CIRCLE_SHA1}-alpine-dev fundocker/marsha:${DOCKER_TAG}-alpine-dev
             docker images "fundocker/marsha:${DOCKER_TAG}*"
+            if [[ ! -z "$CIRCLE_TAG" ]]; then
```

Comment:
the `-z` confuse me in fact. `! -z` can be replace by `-n`

Reply:
:pray:  I'm complete a bash noob lol

Reply:
fixed